.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
- [ ] Please I'm still getting the same error as the last reviewer's notice when the user tries to add ingredients.
![image](https://user-images.githubusercontent.com/74744787/222928610-5cb13b6d-7543-467f-9456-99f55782700a.png)